auslib/blobs/schemas/apprelease-v9.yml

```diff
@@ -101,6 +101,9 @@
               promptWaitTime:
                 type: number
                 description: The number of seconds to wait before showing the dialog which prompts users to restart, overrides the in-app preference
+              disableBITS:
```

Comment:
Nit: is it worth mentioning that despite this being `boolean`, it produces any effects should it's set to `True` only?

Reply:
That's a good point actually. I'll add a note.